data_conventions.yaml

```diff
@@ -56,32 +56,56 @@ standard_tags:
         type: string
         comment: 'Either "client" or "server" for the appropriate roles in an RPC'
         examples: '"client", "server"'
- 
+
     'peer.hostname':
         type: string
         comment: 'Remote hostname'
         examples: '"opentracing.io", "internal.dns.name"'
- 
+
     'peer.ipv4':
         type: string
         comment: 'Remote IPv4 address as a .-separated tuple'
         examples: '"127.0.0.1"'
- 
+
     'peer.ipv6':
         type: string
         comment: 'Remote IPv6 address as a string of colon-separated 4-char hex tuples'
         examples: '"2001:0db8:85a3:0000:0000:8a2e:0370:7334"'
- 
+
     'peer.port':
         type: integer
         comment: 'Remote port'
         examples: '80'
- 
+
     'peer.service':
         type: string
         comment: 'Remote service name (for some unspecified definition of "service")'
         examples: '"elasticsearch", "a_custom_microservice", "memcache"'
- 
+
+    #####################################################################
+    # DATABASE tags
+    #####################################################################
+
+    'db.type':
+        type: string
+        comment: 'Database type'
+        examples: 'For any SQL database, "sql". For others, the lower-case database category, e.g. "cassandra", "hbase", or "redis".'
+
+    'db.instance':
+        type: string
+        comment: 'Database instance name'
+        examples: 'In java, if the jdbc.url="jdbc:mysql://127.0.0.1:3306/customers", instance name is "customers".'
```

Comment:
> it's unlikely to go and parse the string to figure out what the db instance name is.

Make sense to me. And this becomes a tricky problem. 😭😭 `db.url` and `db.instance` are both untouchable because of DataSource.